nav/nav_technique/__init__.py

```
@@ -1,9 +1,9 @@
 from .nav_technique import NavTechnique  # noqa: F401
-from .nav_technique_all_models import NavTechniqueAllModels  # noqa: F401
+from .nav_technique_correlate_all import NavTechniqueCorrelateAll  # noqa: F401
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)